src/client/context/BehandlingerContext.js

```diff
@@ -38,11 +38,15 @@ export const BehandlingerProvider = ({ children }) => {
 
     const fetchBehandlingerMedPersoninfo = async () => {
         const alleBehandlinger = await fetchAlleBehandlinger();
+        setBehandlinger(alleBehandlinger);
+        setValgtBehandling(undefined);
         if (alleBehandlinger !== undefined) {
             const behandlingerMedPersoninfo = await hentNavnForBehandlinger(alleBehandlinger);
-            setBehandlinger(behandlingerMedPersoninfo);
+            if (valgtBehandling === undefined) {
```

Comment:
Det skalerer nok dårlig å sjekke for alt som kan ha skjedd i mellomtiden, men vi kan vel ha det slik inntil navneoppslagene har blitt gjort asynkrone.